bioconda_utils/cli.py

```diff
@@ -98,6 +103,24 @@ def build(recipe_folder,
     else:
         docker_builder = None
 
+    # handle git range
+    if git_range:
+        p = sp.run(['git', 'diff',
+                          '--relative=recipes',
+                          '--name-only'] + git_range +
+                         [os.path.join(recipe_folder, '*', 'meta.yaml'),
```

Comment:
Well, the this version shows in what kind of pattern recipes are expected. Using all files and `endswith`, recipes can be everywhere (within the recipe folder). I find this more implicit. But this is rather an explanation what I had in mind, and why I don't really see the advantage of the other approach. Maybe best would be to have a common recipe discovery for both `--packages *` and `--git-range`. In any case, I think I'd rather wait on Ryans refactorings (see the linting PR), because he has also generalized parts of this code.